Packs/Reco/Integrations/Reco/README.md

```diff
@@ -379,8 +379,8 @@ Get files exposed to a specific email address
 | Reco.Assets.asset | Unknown | Json string of the asset's url and the name | 
 | Reco.Assets.data_category | String | The data category of the asset | 
 | Reco.Assets.data_categories | String | The data categories of the asset |
-| Reco.SensitiveAssets.location | String | The path of the asset. |
-| Reco.SensitiveAssets.source | String | SaaS tool source of the asset. |
```

Comment:
Why are we removing those paths?

Reply:
@maimorag  can you explain here

Reply:
Yeah sure, in the last PR, there was a mismatch between the context path as stated in the yml compared to README, so know @GalNakash-RecoLabs fixed here the correct paths